app/pods/cases/case/subcases/overview/controller.js

```diff
@@ -5,6 +5,8 @@ import { action } from '@ember/object';
 
 export default class CasesCaseSubcasesOverviewController extends Controller {
   @tracked case;
+  @tracked page = 0;
+  @tracked size = 25;
```

Comment:
The JIRA story's description specifies that we should show 20 subcases. I was a bit lazy and set it to 25 (because that's the closes value that's provided by `Auk::Pagination` by default). If showing 20 subcases instead of 25 is desired, this can be changed of course.

Reply:
Doesn't really matter, as long as they can change it. 25 is fine for me